technic/machines/register/cables.lua

```diff
@@ -11,6 +11,10 @@ function technic.get_cable_tier(name)
 	return cable_tier[name]
 end
 
+function technic.register_cable_tier(name, tier)
+	cable_tier[name] = tier
```

Comment:
This to enforce correct API usage early on to inform modders about potential issues.

```suggestion
	assert(technic.machines[tier], "Tier does not exist")
	assert(type(name) == "string", "Invalid node name")

	cable_tier[name] = tier
```